src/youtube-player/youtube-player.ts

```diff
@@ -249,166 +254,122 @@ export class YouTubePlayer implements AfterViewInit, OnDestroy, OnInit {
   seekTo(seconds: number, allowSeekAhead: boolean) {
-    if (!this._player) {
-      return;
+    if (this._player) {
+      this._player.seekTo(seconds, allowSeekAhead);
     }
-    this._player.seekTo(seconds, allowSeekAhead);
   }
 
   /** See https://developers.google.com/youtube/iframe_api_reference#mute */
   mute() {
-    if (!this._player) {
-      return;
+    if (this._player) {
+      this._player.mute();
     }
-    this._player.mute();
   }
 
   /** See https://developers.google.com/youtube/iframe_api_reference#unMute */
   unMute() {
-    if (!this._player) {
-      return;
+    if (this._player) {
+      this._player.unMute();
     }
-    this._player.unMute();
   }
 
   /** See https://developers.google.com/youtube/iframe_api_reference#isMuted */
   isMuted(): boolean {
-    if (!this._player) {
-      return false;
-    }
-    return this._player.isMuted();
+    return !this._player || this._player.isMuted();
   }
 
   /** See https://developers.google.com/youtube/iframe_api_reference#setVolume */
   setVolume(volume: number) {
-    if (!this._player) {
-      return;
+    if (this._player) {
+      this._player.setVolume(volume);
     }
-    this._player.setVolume(volume);
   }
 
   /** See https://developers.google.com/youtube/iframe_api_reference#getVolume */
   getVolume(): number {
-    if (!this._player) {
-      return 0;
-    }
-    return this._player.getVolume();
+    return this._player ? this._player.getVolume() : 0;
   }
 
   /** See https://developers.google.com/youtube/iframe_api_reference#setPlaybackRate */
   setPlaybackRate(playbackRate: number) {
-    if (!this._player) {
-      return;
+    if (this._player) {
+      return this._player.setPlaybackRate(playbackRate);
     }
-    return this._player.setPlaybackRate(playbackRate);
   }
 
   /** See https://developers.google.com/youtube/iframe_api_reference#getPlaybackRate */
   getPlaybackRate(): number {
-    if (!this._player) {
-      return 0;
-    }
-    return this._player.getPlaybackRate();
+    return this._player ? this._player.getPlaybackRate() : 0;
   }
 
   /** See https://developers.google.com/youtube/iframe_api_reference#getAvailablePlaybackRates */
   getAvailablePlaybackRates(): number[] {
-    if (!this._player) {
-      return [];
-    }
-    return this._player.getAvailablePlaybackRates();
+    return this._player ? this._player.getAvailablePlaybackRates() : [];
   }
 
   /** See https://developers.google.com/youtube/iframe_api_reference#getVideoLoadedFraction */
   getVideoLoadedFraction(): number {
-    if (!this._player) {
-      return 0;
-    }
-    return this._player.getVideoLoadedFraction();
+    return this._player ? this._player.getVideoLoadedFraction() : 0;
   }
 
   /** See https://developers.google.com/youtube/iframe_api_reference#getPlayerState */
   getPlayerState(): YT.PlayerState | undefined {
-    if (!window.YT) {
+    if (isBrowser && !window.YT) {
```

Comment:
Just to double-check. It is by intention to return `UNSTARTED` on the server, or should it be undefined as well?

Reply:
Either way makes sense. The main point was to avoid accessing `window`.